acceptance-tests/src/testFixtures/java/tech/pegasys/artemis/test/acceptance/dsl/ArtemisDepositSender.java

```diff
@@ -65,7 +66,7 @@ public String sendValidatorDeposits(final BesuNode eth1Node, final int numberOfV
         "--eth1-endpoint",
         eth1Node.getInternalJsonRpcUrl());
     container.start();
-    Waiter.waitFor(() -> assertThat(container.isRunning()).isFalse());
+    Waiter.waitFor(() -> assertThat(container.isRunning()).isFalse(), TimeUnit.SECONDS, 60);
```

Comment:
I will revert this and see but the `ArtemisDepositSender` is generally flaky here because it does retries inside the container and generally takes around 30s to start up do its thing and shut down.

If the pipeline fails I will put it back in.